admin/data/sql/migrations/090.do.add-bulk-permissions-pupil-table.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const config = require('../../../config')
+
+module.exports.generateSql = function () {
+  return ` GRANT ALTER ON [mtc_admin].[pupil] TO ${config.Sql.Application.Username}`
```

Comment:
Why does the App user need `ALTER TABLE` perms?  Are we dynamically modifying the table during the application run?

Reply:
Bulk insert requires Alter table permissions on pupil table with our current db user.
The query will fail if they are not supplied.

Reply:
Ah, of course.